machine/machine_core/machine.py

```diff
@@ -53,8 +53,8 @@
 
 
 class Machine(ssh_connection.SSHConnection):
-    def __init__(self, address="127.0.0.1", image="unknown", verbose=False, label=None, browser=None,
-                 user="root", identity_file=None, arch="x86_64", ssh_port=22, web_port=9090):
+    def __init__(self, address="127.0.0.1", image="unknown", verbose=False, console_file=None, label=None,
+                 browser=None, user="root", identity_file=None, arch="x86_64", ssh_port=22, web_port=9090):
 
```

Comment:
This is the wrong place. The *only* class where we can do this is VirtMachine. We can't control already running/external/Testing Farm etc. machine types. Please move all of this into VirtMachine and handle the argument there. Machine and SshConnection should not be touched by this.